src/components/reserveCar.jsx

```diff
@@ -0,0 +1,85 @@
+import React, { useState } from "react";
+import { useSelector, useDispatch } from "react-redux";
+import { useNavigate } from "react-router-dom";
+import { createNewReservation } from "../store/reservationReducer";
+import "./ReserveCar.css";
+
+function ReserveCar() {
+  const dispatch = useDispatch();
+  const navigate = useNavigate();
+  const car = useSelector((state) => state.cars.car);
+  console.log("car", car);
```

Comment:
- [ ] Kindly delete this old piece of debugging console.log code, to improve the cleanliness of the repo. 👍